### PR TITLE
Added title and made it dynamic with input #132

### PR DESCRIPTION
**Pull request by jumankaushik**
### Issue #132 

- Added the title on the Confirm.js page according to input value.

image

Result: 

image

### CODE REVIEW

File: src/components/Confirm.js [hunk 1]
```diff
@@ -174,6 +174,7 @@ function ConfirmationCheckbox() {
           }}
         >
           <Confetti></Confetti>
+          <h2 style={{ color: "black", padding: "10px" }}>{inputValue}, your future wife is</h2>
```
Comment (1Shubham7) [inline src/components/Confirm.js:177]: please make the text color white or some other color, black isn't cool : )

File: src/components/bestbutton.css [hunk 2]
```diff
@@ -70,4 +70,20 @@
 .bestbutton:hover i::after{
     width : 20px;
     left: 80%;
+}
```
Comment (1Shubham7) [inline src/components/bestbutton.css:73]: remove this part, the PR should only be about showing text.

**Comment by 1Shubham7** [comment]
@jumankaushik always remember that one PR must only solve one issue, it is difficult to review a PR doing two things, please keep this PR about adding that text, edit the title and discription accordingly : )

**Comment by jumankaushik** [comment]
Okay

**Comment by jumankaushik** [comment]
Done. Kindly check!

**Comment by 1Shubham7** [comment]
Thanks for your contribution @jumankaushik

**Pull request merged**